src/java/voldemort/server/protocol/admin/AdminServiceRequestHandler.java

```diff
@@ -496,8 +496,8 @@ public StreamRequestHandler handleRequest(final DataInputStream inputStream,
             for(String storeName: storeNames) {
 
                 ReadOnlyStorageEngine store = getReadOnlyStorageEngine(metadataStore,
-                                                                       storeRepository,
```

Comment:
Though it is OK for now, you should have an editor setting format only editted lines. It makes it harder to review on big changes.